prime-dotnet-webapi/Startup.cs

```diff
@@ -76,13 +76,18 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IPartyService, PartyService>();
             services.AddScoped<IPdfService, PdfService>();
             services.AddScoped<IPlrProviderService, PlrProviderService>();
-            services.AddScoped<IPrivilegeService, PrivilegeService>();
-            services.AddScoped<IRazorConverterService, RazorConverterService>();
-            services.AddScoped<ISiteService, SiteService>();
-            services.AddScoped<ISoapService, SoapService>();
-            services.AddScoped<ISubmissionService, SubmissionService>();
-            services.AddScoped<ISubmissionRulesService, SubmissionRulesService>();
+            services.AddScoped<IHealthAuthorityService, HealthAuthorityService>();
+            services.AddScoped<IEmailTemplateService, EmailTemplateService>();
+            services.AddScoped<IAuthorizedUserService, AuthorizedUserService>();
+            services.AddScoped<IEnrolleePaperSubmissionService, EnrolleePaperSubmissionService>();
+            services.AddScoped<IOrganizationClaimService, OrganizationClaimService>();
             services.AddScoped<IVerifiableCredentialService, VerifiableCredentialService>();
+            services.AddScoped<ISubmissionRulesService, SubmissionRulesService>();
+            services.AddScoped<ISubmissionService, SubmissionService>();
+            services.AddScoped<ISoapService, SoapService>();
+            services.AddScoped<ISiteService, SiteService>();
+            services.AddScoped<IRazorConverterService, RazorConverterService>();
+            services.AddScoped<IPrivilegeService, PrivilegeService>();

```

Comment:
I alphabetized these services in a PR yesterday, which is why you are seeing this diff.

Reply:
I'll resolve conflicts